src/vk/VkColor.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Vulkan
+{
+    [StructLayout(LayoutKind.Sequential)]
+    public struct VkColor4
```

Comment:
I think I'd rather leave these new custom structs out of the library, especially with their current naming (makes it sound like they are actually a part of Vulkan).